server/api/follower_handler.py

```diff
@@ -0,0 +1,66 @@
+from flask import jsonify, Blueprint
+from models.user import User
+from database import db
+from api.auth_handler import token_getter
+
+follower_handler = Blueprint('follower_handler', __name__)
+
+
+@follower_handler.route('/followers/<user_id>', methods=['GET'])
```

Comment:
This should be `GET /followers`. You don't need the user_id since it's in the token